package.json

```diff
@@ -37,6 +37,7 @@
     "compute-baseline": "file:./packages/compute-baseline",
     "fast-json-stable-stringify": "^2.1.0",
     "fdir": "^6.1.1",
+    "marked": "^12.0.1",
```

Comment:
`marked` is not very compliant with CommonMark. While we're especially unlikely to exercise any of Markdown's "fun" bits, I'd prefer to choose a good dependency _now_, in case we start using Markdown for other things. May I suggest `remark` or `micromark` instead?

Reply:
I picked `marked` because it has no dependencies, but I've never used any of these libraries before so I'm not opinionated. Is there one that you like, and do you know if any of them make it easy to produce plain text without parsing the resulting HTML?

Reply:
`remark` is the thing to use then. It won't _look_ very dependency-light (the "unified" project is made up of a bunch of tiny packages that transform markdown/html/etc., though they're mostly developed together), but it's nice enough to join up `remark` + [`remark-html`](https://unifiedjs.com/explore/package/remark-html/#use) or [`strip-markdown`](https://unifiedjs.com/explore/package/strip-markdown/#use) get different outputs from the same parsed Markdown.

(If that sounds like a hassle, I've done this sort of thing before and I can open a competing PR to demo that.)

Reply:
I've pushed some changes.